account_due_list/init_hook.py

```diff
@@ -0,0 +1,121 @@
+# -*- coding: utf-8 -*-
+# © 2016 David Dufresne <david.dufresne@savoirfairelinux.com>
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+import logging
+
+from openerp import SUPERUSER_ID
+from openerp.api import Environment
+from openupgradelib import openupgrade
```

Comment:
Don't use openupgradelib for other things except migrations. Anyway, loggued_query is not very useful in this case.